mscore/pagesettings.cpp

```diff
@@ -97,10 +97,10 @@ void PageSettings::hideEvent(QHideEvent* ev)
 //   setScore
 //---------------------------------------------------------
 
-void PageSettings::setScore(MasterScore* s)
+void PageSettings::setScore(Score* s)
       {
-      cs  = s;
-      MasterScore* sl = s->clone();
+      cs = static_cast<MasterScore*>(s);
```

Comment:
Are we really sure that `s` is a `MasterScore` here before casting it? 
If we are, why not keep on passing a MasterScore ?

Reply:
PageSettings is initiated from musescore.cpp where the call used to be setScore(cs), with cs  of type MasterScore. This has changed from version 2.2 to current master, so cs is now of type Score.
Building gives compile error:
error C2664: 'void Ms::PageSettings::setScore(Ms::MasterScore *)': cannot convert argument 1 from 'Ms::Score *' to 'Ms::MasterScore *'.

However since all scores are created as type MasterScore,  cs is in fact pointing to a MasterScore object, which I believe makes it safe to change PageSetttings/setScore to take a Score* argument, and then do the static_cast.

I don't think that the change made in musscore.cpp should be reverted, at least not without a clear understanding why it was made.

Alternatively we could remove the need for the cast, wich is the use of the clone() method. This is a member of MasterScore, and not Score. It might be a possibility to move the method to Score, but that decision should be taken by someone with more knowledge and insight than me.